src/App.scss

```diff
@@ -7,6 +7,9 @@
 
 $colord: #277a9f;
 $text: #647B83;
+$colorm: #37b0e7;
+$colorl: #EFF4F5;
```

Comment:
These two new variables, along with the existing variable `$colord` define a sort of "color scheme" (which, by the way, is [borrowed from some of our other existing sites](https://github.com/brigadecore/brigade/blob/main/docs/assets/scss/brigade.scss#L11-L13)).

If you could please move these two new variables up _between_ the existing `$colord` and `$text` variables, the three colors will be grouped together and it will be, visually, a little more clear that the three colors form a scheme.